common/djangoapps/student/views.py

```diff
@@ -2098,7 +2099,7 @@ def password_reset(request):
     form = PasswordResetFormNoActive(request.POST)
     if form.is_valid():
         form.save(use_https=request.is_secure(),
-                  from_email=microsite.get_value('email_from_address', settings.DEFAULT_FROM_EMAIL),
+                  from_email=theming_helpers.get_value('email_from_address', settings.DEFAULT_FROM_EMAIL),
```

Comment:
I think it is ok, you can see old implementation was also using the same. I just replace the `microsite` with `theming_helpers`